Readme.md

```diff
@@ -194,6 +198,21 @@ fn("/invalid"); //=> false
 fn("/user/caf%C3%A9"); //=> { path: '/user/caf%C3%A9', index: 0, params: { id: 'café' } }
 ```
 
+The `match` function can be used to custom match named parameters. For example, this can be used to whitelist a small number of valid paths:
```

Comment:
```suggestion
The `match` function can be used with custom match named parameters. For example, this can be used to whitelist a small number of valid paths:
```

Is the goal of this section just to demonstrate you can use all the same paths as the previous documentation sections? Would it be better just to say that explicitly?

Reply:
The goal is mostly just to demonstrate how the lib could be used to name some matched parameters. With the existing docs, it took me a while to understand that `:tab(about|photos)` would actually provide a `tab` param with the matched value.